src/pages/UserAdmin/index.tsx

```diff
@@ -65,6 +65,7 @@ const UserAdmin = () => {
   }, [tab, userClassChoice, userStudyChoice, searchInput]);
   const { data, error, hasNextPage, fetchNextPage, isLoading, isFetching } = useUsers({ is_TIHLDE_member: tab === membersTab.value, ...filters });
   const isEmpty = useMemo(() => (data !== undefined ? !data.pages.some((page) => Boolean(page.results.length)) : false), [data]);
+  const members = data !== undefined ? data.pages.map((page) => page.results.length) + ' medlemmer' : '';
```

Comment:
Antall brukere totalt kan være mer enn bare dem som er på siden som sees (`page.results`). Du kan finne totalt antall som backend har på nåværende filtrering (klasse/studie) med `page.count`. I tillegg tror jeg det er lurt å vise hvilken filtering som gjelder eksplisitt så alle forstår hvilket antall en ser. Tror denne teksten skal funke:
```suggestion
  const membersAmount = `${data?.pages[0]?.count || '_'} medlemmer i ${userClassChoice !== 0 ? USER_CLASSES[userClassChoice] : ''} ${userStudyChoice !== 0 ? USER_STUDIES[userStudyChoice] : ''}`;
```